eng/versioning/external_dependencies.txt

```diff
@@ -180,7 +180,7 @@ cosmos_io.netty:netty-handler;4.1.45.Final
 cosmos_io.netty:netty-handler-proxy;4.1.45.Final
 cosmos_io.netty:netty-tcnative;2.0.27.Final
 cosmos_io.netty:netty-transport-native-epoll;4.1.45.Final
-cosmos_io.projectreactor:reactor-core;3.3.2.RELEASE
+cosmos_io.projectreactor:reactor-core;3.3.3.RELEASE
```

Comment:
@alzimmermsft @kushagraThapar 
Right now, with these changes, every cosmos_io.* variable is now duplicated since the the common versions of these external dependencies has been brought forward. The cosmos* variables, that are duplicates, should no longer be here.

Reply:
Cosmos is currently investigating an issue where they would like to continue using their own dependency set, maybe this could be unified once Cosmos releases their GA'd library.

@kushagraThapar

Reply:
That's fine for the meantime and I'll approve the PR but please file an issue for Cosmos to remove the cosmos_* dependencies that are the same version as the common dependency versions before they GA.